mscore/plugin/api/part.h

```diff
@@ -25,10 +25,32 @@ namespace PluginAPI {
 
 class Part : public Ms::PluginAPI::ScoreElement {
       Q_OBJECT
-      Q_PROPERTY(int                            startTrack          READ startTrack)
-      Q_PROPERTY(int                            endTrack            READ endTrack)
+      Q_PROPERTY(int                            startTrack           READ startTrack)
+      Q_PROPERTY(int                            endTrack             READ endTrack)
       /// The string identifier for the current instrument. \since MuseScore 3.2
-      Q_PROPERTY(QString                        instrumentId        READ instrumentId)
+      Q_PROPERTY(QString                        instrumentId         READ instrumentId)
+      /// TBD \since MuseScore 3.2
```

Comment:
Why TBD? harmonyCount is the number of Chord Symbols

Reply:
Also the 3.2 version should be released very soon so this PR won't be able to get to that version. Could you also please change these `\since` statements for the new properties to `\since MuseScore 3.2.1`?

Reply:
>Why TBD? harmonyCount is the number of Chord Symbols for example.
>The others should be equally easy to document I think?

You're mistaking me for an expert on the subject matter when I'm not. Code on the other hand is something I know. I'm just learning the music stuff. Hence I don't wish to make assumptions about them.

For instance, until you stated this about harmonyCount I didn't know that was the definition. I would have said it's the number of harmonies. ;)

If someone could fill out the rest I'll add them. Bad information can be worse than no info from my perspective.

-Dale

Reply:
>Could you also please change these \since statements for the new properties to \since MuseScore 3.2.1?

Ok.

-Dale

Reply:
You can look them up in the code.

Reply:
>You can look them up in the code.

Believe me I tried. To the extent I even started digging through MusicXML specs to glean the concepts I'm missing.

-Dale